### DISCUSSION TITLE
Main Bumpkin Girl copyright alert

### DISCUSSION THREAD

**Discussion by reda601** [sunflower-land/sunflower-land#1385] [Art] [answered]
I am not sure who's the owner of this pixel art, but the exact same bumpkin girl you use in the game is used as well in a mobile game called **THE WAY HOME**, I'll join the screenshots of the game in playstore and the the pic of the bumpkin in the game down and you can see it as well in the game's playstore pictures gallery.
 
![name](https://user-images.githubusercontent.com/83256204/187770423-2f0d1282-240c-4624-bd4d-4b8d27660cfa.jpeg)
![pic](https://user-images.githubusercontent.com/83256204/187770450-7e3a73e4-cdc4-4449-a53c-afd6fcc7687c.jpeg)

**Comment by adamhannigan** [comment, marked as answer]
@reda601 Thanks for raising this.

We extended a base asset pack that we purchased that includes a few characters, trees and the crops. That female character is included in the pack. As long as the artist purchased the pack, they have the licence to use that asset as well 😄 

You have a keen eye though!

**Reply by reda601** [reply]
thanks for your answer.
